src/library/objects/Messengers.js

```diff
@@ -24,7 +24,7 @@ Sample Usage:
 (function(){
 	"use strict";
 	
-	console.info("KC3改 Messengers loaded");
+	console.log("KC3改 Messengers loaded");

```

Comment:
ah, current grunt remove logging default config will remove levels including info, warn... that is confirmed, and why I added the mark `/*RemoveLogging:skip*/` to some critical places